frontends/web/src/routes/exchange/components/buysell.tsx

```diff
@@ -105,6 +105,41 @@ export const BuySell = ({
     <>
       <div className={style.innerRadioButtonsContainer}>
         {!exchangeDealsResponse && <Skeleton />}
+
+        {action === 'sell' ? (
+          <div key={'btcdirect-sell'} className={style.actionableItemContainer}>
+            ----START Testing: BTC Direct sell
```

Comment:
Keeping this for now, so I can't miss and accidentally continue with the buy widget.